### ISSUE TITLE
Puppeteer actions not working / possible

### ISSUE THREAD

**Issue by pbernasconi** [modelcontextprotocol/servers#53]
I'm pretty sure I've setup everything correctly - but when I run a simple command like `open paolobernasconi.com in puppeteer` - claude tells me it's not possible.

Here are my logs showing the successful MCP server connection:

![CleanShot 2024-11-25 at 21 44 30](https://github.com/user-attachments/assets/4412a3cb-0b6b-4c84-96d7-a941f34f613b)

And here are some prompts I've been trying:

![CleanShot 2024-11-25 at 21 45 04](https://github.com/user-attachments/assets/03859fc9-ad24-4d01-9266-2dc950b6f257)

**Comment by planeyang** [comment]
I have a similar issue and my Claude desktop complains that it cannot connect to puppeteer MCP server

**Comment by pbernasconi** [comment]
Yup ` npx @modelcontextprotocol/server-puppeteer` works (waiting for input)

![CleanShot 2024-11-26 at 09 19 00](https://github.com/user-attachments/assets/6b9624fb-48a7-41a6-a500-162dc1f3a46b)


My node version  is `v18.18.2`

**Comment by pbernasconi** [comment]
Ohhh weird: just tried and now it's working - no idea what changed overnight, I didn't edit anything!

![CleanShot 2024-11-26 at 09 20 46](https://github.com/user-attachments/assets/324f6d0d-7bc0-4e05-99a1-958d0788cbf6)

**Comment by pbernasconi** [comment]
works great - except after the 1st website visit, there's a detatched frame issue that can't be fixed without quitting Claude app and trying again.

![CleanShot 2024-11-26 at 09 25 30](https://github.com/user-attachments/assets/ab81a588-76f1-41e0-8d79-8d8441782b89)

**Comment by dgormly** [comment]
I get the same failed call

**Comment by shelmin** [comment]
I encountered the same issue. Is there a solution about this?

**Comment by mdfu** [comment]
I'm also getting this on Krister. Initially works, and then for some reason after the first use, subsequent use it comes up with this error. 

("error","WCP error -32603: Attempted to use detached Frame
944EC83495B7792A45CD0F8AD4D812",")

**Comment by sag1v** [comment]
happens with cursor as well. Initially works but then getting the detached error
> "error","WCP error -32603: Attempted to use detached Frame
